packages/transition-backend/src/models/db/transitSchedules.db.queries.ts

```diff
@@ -800,6 +800,39 @@ const duplicateSchedule = async ({
     }
 };
 
+const getTripsInTimeRange = async ({
+    rangeStart,
```

Comment:
Please specifiy format as comment (epoch, time from midnight? Second, minutes?)

Reply:
Probably also add in the function comment header describing the feature that the time in the range is inclusive